src/terraform-provider-signalform/signalform/provider.go

```diff
@@ -28,7 +28,7 @@ func Provider() terraform.ResourceProvider {
 			"auth_token": &schema.Schema{
 				Type:        schema.TypeString,
 				Optional:    true,
-				DefaultFunc: schema.EnvDefaultFunc("SFX_AUTH_TOKEN", ""),
+				DefaultFunc: schema.EnvDefaultFunc("SFX_AUTH_TOKEN", nil),
```

Comment:
won't `length(nil)` blow up in case `AuthToken` is not set (which is likely what's going to happen)?

Reply:
I've not seen any issues here in `make test`. Presumably `len(nil)` == 0?